libr/flag/flag.c

```diff
@@ -331,24 +331,44 @@ static bool print_flag_json(RFlagItem *flag, void *user) {
 
 static bool print_flag_rad(RFlagItem *flag, void *user) {
 	struct print_flag_t *u = (struct print_flag_t *)user;
+	char *comment_b64 = NULL, *tmp = NULL;
 	if (u->in_range && (flag->offset < u->range_from || flag->offset >= u->range_to)) {
 		return true;
 	}
 	if (!u->fs || flag->space != u->fs) {
 		u->fs = flag->space;
 		u->f->cb_printf ("fs %s\n", u->fs? u->fs->name: "*");
 	}
+	if (flag->comment && *flag->comment) {
+		comment_b64 = r_base64_encode_dyn(flag->comment, -1);
+		// prefix the armored string with "base64:"
+		if (comment_b64) {
+			tmp = malloc(strlen(comment_b64) + 8);
```

Comment:
use r_str_newf ("base64:%s", comment_b64); instead of messing with malloc+strlen+strcpy+memcpy